Cannot install behind a corporate proxy

Hi! I'm trying to install this module using `npm i --save-dev @apollo/rover` and I have this:

```
npm ERR! code 1
npm ERR! path C:\Users\pplenkov\Documents\git_projects\central-deploy-xsa\gitlab\node_modules\@apollo\rover
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node ./install.js
npm ERR! Downloading release from https://github.com/apollographql/rover/releases/download/v0.3.0/rover-v0.3.0-x86_64-pc-windows-msvc.tar.gz
npm ERR! If you would like to disable Rover's anonymized usage collection, you can set APOLLO_TELEMETRY_DISABLED=1
npm ERR! You can check out our documentation at https://go.apollo.dev/r/docs.
npm ERR! Debugger attached.
npm ERR! Error fetching release: write EPROTO 7528:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:c:\ws\deps\openssl\openssl\ssl\record\ssl3_record.c:332:
```

Debug indicates that it's 100% a proxy issue. We have HTTP_PROXY and HTTPS_PROXY variables maintained..  It looks like issue is inherited from binary-install module. The possible solution may be to maintain a global https agent in install.js.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -18,6 +18,60 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## 📚 Documentation -->
 
+# [0.5.0] - 2022-04-11
+
+> Important: X breaking changes below, indicated by **❗ BREAKING ❗**
+
+## ❗ BREAKING ❗
+
+- **`rover-fed2` has been deprecated - @EverlastingBugstopper, #1058**
+
+  `rover fed2 supergraph compose` has been deprecated. You should instead set `federation_version: 2` in your `supergraph.yaml` to use Federation 2 with the `rover supergraph compose` command.
+
+## 🚀 Features
+
+- **`rover supergraph compose` optionally updates automatically - @EverlastingBugstopper, #1058 fixes #2046**
+
+  When running `rover supergraph compose`, Rover will automatically download the correct version of composition to use. In your `supergraph.yaml` files, you can specify `federation_version: 1` or `federation_version: 2` to always get the latest updates. You can pass the `--skip-update` flag to skip checking for an update. You can also specify an exact version if you'd like to pin your federation version, like so: `federation_version: =2.0.0`.
+
+  Additionally, you can run `rover install --plugin supergraph@latest-2` or `rover install --plugin supergraph@v2.0.0` to install a plugin ahead of time, which may be helpful in CI. For Federation 2, you'll have to accept the ELv2 license one time per machine. You likely want to set `APOLLO_ELV2_LICENSE=accept` in CI if you are using Federation 2.
+
+- **Adds `--insecure-unmask-key` to `rover config whoami` - @EverlastingBugstopper, #1043 fixes #1023**
+
+  Previously, running `rover config whoami` would output your entire API key to the terminal. This is not the documented behavior, and it is insecure because someone could be sharing their screen while trying to debug and accidentally leak their API key.
+
+  Now, `rover config whoami` will mask your API key when it prints to the terminal. You can override this behavior by passing the `--insecure-unmask-key` flag.
+
+- **Retry on timeouts and connection errors - @ptondereau, #1014 fixes #790**
+
+  Rover will now automatically retry HTTP requests that fail due to timeouts or initial connection errors.
+
+- **Define an HTTP agent for non-studio requests - @ptondereau, #1075 fixes #961**
+
+  Rover now sends a User-Agent header along with all requests, not just requests to Apollo Studio.
+
+- **Adds support for HTTP(S) proxies in npm installer - @farawaysouthwest, #1067 fixes #899**
+
+  You can now install Rover from npm if you are behind a proxy.
+
+## 🐛 Fixes
+
+- **Fixed a dead link in ARCHITECTURE.md - @ptondereau, #1053**
+
+## 🛠 Maintenance
+
+- **Simplify `rover subgraph fetch` query - @EverlastingBugstopper, #1056 fixes #992**
+
+  `rover subgraph fetch` now uses a much more efficient query that only requests a single subgraph at a time rather than all of them. Yay GraphQL!
+
+- **Upgrades `apollo-encoder` - @bnjjj, #1017 fixes #1010**
+
+## 📚 Documentation
+
+- **Set up new docs infrastructure - @trevorblades, #1051, #1052**
+
+  @trevorblades has done an awesome job setting up new docs for Apollo, including Rover! Check out the [shiny new repo](https://github.com/apollographql/docs).
+
 # [0.5.0-rc.1] - 2022-04-05
 
 ## 🐛 Fixes

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,7 +11,7 @@ license-file = "./LICENSE"
 name = "rover"
 readme = "README.md"
 repository = "https://github.com/apollographql/rover/"
-version = "0.5.0-rc.1"
+version = "0.5.0"
 default-run = "rover"
 
 publish = false

--- a/README.md
+++ b/README.md
@@ -37,7 +37,7 @@ rover graph publish --schema ./path-to-valid-schema test@cats
 ## Command-line options
 
 ```console
-Rover 0.5.0-rc.1
+Rover 0.5.0
 
 Rover - Your Graph Companion
 Read the getting started guide by running:
@@ -142,7 +142,7 @@ To install a specific version of Rover (note the `v` prefixing the version numbe
 > Note: If you're installing Rover in a CI environment, it's best to target a specific version rather than using the latest URL, since future major breaking changes could affect CI workflows otherwise.
 
 ```bash
-curl -sSL https://rover.apollo.dev/nix/v0.5.0-rc.1 | sh
+curl -sSL https://rover.apollo.dev/nix/v0.5.0 | sh
 ```
 
 You will need `curl` installed on your system to run the above installation commands. You can get the latest version from [the curl downloads page](https://curl.se/download.html).
@@ -160,7 +160,7 @@ To install a specific version of Rover (note the `v` prefixing the version numbe
 > Note: If you're installing Rover in a CI environment, it's best to target a specific version rather than using the latest URL, since future major breaking changes could affect CI workflows otherwise.
 
 ```bash
-iwr 'https://rover.apollo.dev/win/v0.5.0-rc.1' | iex
+iwr 'https://rover.apollo.dev/win/v0.5.0' | iex
 ```
 
 #### npm installer

--- a/docs/source/getting-started.md
+++ b/docs/source/getting-started.md
@@ -18,7 +18,7 @@ To install a **specific version** of Rover (recommended for CI environments to e
 
 ```bash
 # Note the `v` prefixing the version number
-curl -sSL https://rover.apollo.dev/nix/v0.5.0-rc.1 | sh
+curl -sSL https://rover.apollo.dev/nix/v0.5.0 | sh
 ```
 
 If your machine doesn't have the `curl` command, you can get the latest version from the [`curl` downloads page](https://curl.se/download.html).
@@ -37,7 +37,7 @@ To install a **specific version** of Rover (recommended for CI environments to e
 
 ```bash
 # Note the `v` prefixing the version number
-iwr 'https://rover.apollo.dev/win/v0.5.0-rc.1' | iex
+iwr 'https://rover.apollo.dev/win/v0.5.0' | iex
 ```
 
 ### `npm` installer

--- a/installers/binstall/scripts/nix/install.sh
+++ b/installers/binstall/scripts/nix/install.sh
@@ -20,7 +20,7 @@ BINARY_DOWNLOAD_PREFIX="https://github.com/apollographql/rover/releases/download
 # Rover version defined in root cargo.toml
 # Note: this line is built automatically
 # in build.rs. Don't touch it!
-PACKAGE_VERSION="v0.5.0-rc.1"
+PACKAGE_VERSION="v0.5.0"
 
 download_binary_and_run_installer() {
     downloader --check

--- a/installers/binstall/scripts/windows/install.ps1
+++ b/installers/binstall/scripts/windows/install.ps1
@@ -14,7 +14,7 @@
 # version found in Rover's Cargo.toml
 # Note: this line is built automatically
 # in build.rs. Don't touch it!
-$package_version = 'v0.5.0-rc.1'
+$package_version = 'v0.5.0'
 
 function Install-Binary($rover_install_args) {
   $old_erroractionpreference = $ErrorActionPreference

--- a/installers/npm/package.json
+++ b/installers/npm/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@apollo/rover",
-  "version": "0.5.0-rc.1",
+  "version": "0.5.0",
   "description": "The new Apollo CLI",
   "main": "index.js",
   "bin": {

